services/headless-lms/migrations/20241201122945_create_privacy_links.up.sql

```diff
@@ -0,0 +1,23 @@
+-- Add up migration script here
+CREATE TABLE privacy_links (
+  id UUID DEFAULT uuid_generate_v4() PRIMARY KEY,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  deleted_at TIMESTAMP WITH TIME ZONE,
+  link_title VARCHAR(255) NOT NULL,
+  link_url TEXT NOT NULL,
+  course_id UUID NOT NULL REFERENCES courses(id),
+  CONSTRAINT unique_privacy_link_course UNIQUE (course_id)
+);
+
+CREATE TRIGGER set_timestamp BEFORE
+UPDATE ON privacy_links FOR EACH ROW EXECUTE PROCEDURE trigger_set_timestamp();
+
+COMMENT ON TABLE privacy_links IS 'This table stores privacy link data associated with specific courses.';
```

Comment:
This comment should describe that there are links in the footer and adding rows to this table will change those links for a specific course.